db/migrate/20240909081233_create_spree_stripe_express_checkout_source.rb

```diff
@@ -0,0 +1,10 @@
+class CreateSpreeStripeExpressCheckoutSource < ActiveRecord::Migration[7.1]
+  def change
+    create_table :spree_stripe_express_checkout_sources do |t|
+      t.string :payment_intent_id
+      t.string :payment_intent_secret
+      t.integer :payment_method_id
+      t.timestamps
+    end
+  end
+end
```

Comment:
were we not able to save payment informations in existing table structures of spree?
Given that it already supports lots of payment gateways already..

My first assumption would be that we wont need to create new table to store payment information